Updated description of how to get the macOS version

madsnedergaard opened the pull request:
Add the release files to 2.0 release on github first :)

Packaged macOS files are currently available here: https://we.tl/PLhyHCyiKp

madsnedergaard commented:
Forgot to mention, this closes #38

rvpanoz commented:
Hello! Yes but I need a new build, because I pushed an updated  version yesterday late at night.. 💯 

When you have time... :)

madsnedergaard commented:
All done, had to add a dependency in order to build it this time though. Added to this repo (also fixed merged conflict in readme). The macOS release files are available here: https://we.tl/yAl5fsNThJ
